ocis-pkg/config/defaults/paths.go

```diff
@@ -40,3 +40,36 @@ func BaseDataPath() string {
 		return ""
 	}
 }
+
+var (
+	// switch between modes
+	BaseConfigPathType = "homedir" // or "path"
```

Comment:
We should consider using build constraints to switch those: see https://stackoverflow.com/questions/36703867/golang-preprocessor-like-c-style-compile-switch and  https://pkg.go.dev/go/build#hdr-Build_Constraints